tests/helpers/linux/test_nodemask.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) ByteDance, Inc. and its affiliates.
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from pathlib import Path
+
+from drgn.helpers.linux.nodemask import (
+    for_each_node,
+    for_each_online_node,
+)
+from tests.helpers.linux import LinuxHelperTestCase
+
+NODE_PATH = Path("/sys/devices/system/node")
+
+def parse_nodelist(nodelist):
+    nodes = set()
+    for node_range in nodelist.split(","):
+        first, sep, last = node_range.partition("-")
+        if sep:
+            nodes.update(range(int(first), int(last) + 1))
+        else:
+            nodes.add(int(first))
+    return nodes
+
+class TestNodeMask(LinuxHelperTestCase):
```

Comment:
Looks like this needs a `@unittest.skipUnless(os.path.exists("/sys/devices/system/node"), "kernel does not support NUMA")`.

Reply:
Yes, I will add this and resend the patch. BTW, Is NUMA node not enabled on your CI test machine?

Reply:
No, it uses a minimal kernel configuration so it appears that it left out NUMA. I'll enable it for future builds, but we should still have this check.